test/compile_errors.zig

```diff
@@ -244,4 +244,18 @@ pub fn addCases(ctx: *Cases, b: *std.Build) !void {
             ":1:5: error: expected expression, found 'invalid token'",
         });
     }
+
+    {
+        const case = ctx.obj("usage of deprecated code", b.graph.host);
+
+        case.addError(
+            \\const bad = @deprecated(42);
+            \\
+            \\pub export fn foo() usize {
+            \\   return bad; 
+            \\}
+        , &[_][]const u8{
+            ":1:13: error: found deprecated code",
+        });
+    }
```

Comment:
This file *should* have a big comment at the top saying "DON'T PUT THINGS HERE" :P

Please put this test in `test/cases/compile_errors/deprecated.zig` (or something like that) instead.